var/spack/repos/builtin/packages/lvarray/package.py

```diff
@@ -38,35 +38,47 @@ class Lvarray(CMakePackage, CudaPackage):
     homepage = "https://github.com/GEOSX/lvarray"
     git      = "https://github.com/GEOSX/LvArray.git"
 
-    version('develop', branch='develop', submodules='True')
-    version('main', branch='main', submodules='True')
-    version('0.1.0', tag='v0.1.0', submodules='True')
+    maintainers = ['corbett5']
+
+    version('develop', branch='develop', submodules=False)
+    version('main', branch='main', submodules=False)
+    version('0.2.1', tag='v0.2.1', submodules=False)
+    version('0.1.0', tag='v0.1.0', submodules=True)
 
     variant('shared', default=True, description='Build Shared Libs')
     variant('umpire', default=False, description='Build Umpire support')
     variant('chai', default=False, description='Build Chai support')
     variant('caliper', default=False, description='Build Caliper support')
+    variant('pylvarray', default=False, description='Build Python support')
     variant('tests', default=True, description='Build tests')
     variant('benchmarks', default=False, description='Build benchmarks')
     variant('examples', default=False, description='Build examples')
     variant('docs', default=False, description='Build docs')
     variant('addr2line', default=True,
             description='Build support for addr2line.')
 
-    depends_on('cmake@3.8:', type='build')
-    depends_on('cmake@3.9:', when='+cuda', type='build')
+    depends_on('blt', when='@0.2.0:', type='build')
+
+    depends_on('camp')
+    depends_on('camp+cuda', when='+cuda')
 
     depends_on('raja')
     depends_on('raja+cuda', when='+cuda')
 
+    # At the moment Umpire doesn't support shared when building with CUDA.
     depends_on('umpire', when='+umpire')
-    depends_on('umpire+cuda', when='+umpire+cuda')
+    depends_on('umpire+cuda~shared', when='+umpire+cuda')
 
     depends_on('chai+raja', when='+chai')
     depends_on('chai+raja+cuda', when='+chai+cuda')
 
     depends_on('caliper', when='+caliper')
 
+    depends_on('python +shared +pic', when='+pylvarray')
+    depends_on('py-numpy@1.19:', when='+pylvarray')
+    depends_on('py-scipy@1.5.2:', when='+pylvarray')
+    depends_on('py-pip', when='+pylvarray')
```

Comment:
Should these be `type=('build', 'link')`?

Reply:
I thought that was the default type?
https://spack.readthedocs.io/en/latest/packaging_guide.html#dependency-types

Reply:
Sorry, I meant `type=('build', 'run')`. Python libraries don't usually have much to link to, but they are often needed in `PYTHONPATH` at run-time.

Reply:
Well we do link to both Python and NumPy. But SciPy doesn't have a C API so we do just use it through Python. So I guess Python and NumPy should be `type=('build', 'link', 'run')` and SciPy and pip should be `type=('build', 'run')`?

Reply:
I'm not even sure if pip is needed, but if it is it's prob only at build time

Reply:
LvArray doesn't actually use pip but when you build Python if you don't include it it's a pain to add. I can remove it if you'd like.

Reply:
I made the type changes and I removed pip.